<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1256</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* ========== Global Base Styling ========== */
QMainWindow, QWidget, QStackedWidget {
    background-color: #1C1E21;  /* eerie-black */
    color: #EBEBEB;             /* antiflash-white */
    font-size: 14px;
}

/* ========== Frame-Like Widgets (e.g. sidebar) ========== */
QFrame {
    background-color: #26282C;  /* raisin-black */
}

/* ========== Buttons ========== */
QPushButton {
    background-color: #2F3237;  /* onyx */
    color: #EBEBEB;
    border: none;
    padding: 6px 12px;
    border-radius: 4px;
}
QPushButton:hover {
    background-color: #3a3d43;
}
QPushButton:pressed {
    background-color: #5196F5;  /* chefchaouen-blue */
    color: #1C1E21;             /* eerie-black */
}

/* ========== Tool Buttons (e.g., toggle button) ========== */
QToolButton {
    background-color: #2F3237;
    color: #EBEBEB;
    border: none;
    padding: 6px;
    border-radius: 3px;
}
QToolButton:hover {
    background-color: #3a3d43;
}
QToolButton:pressed {
    background-color: #5196F5;
    color: #1C1E21;
}

/* ========== Menu Bar ========== */
QMenuBar {
    background-color: #26282C;
    color: #EBEBEB;
}
QMenuBar::item:selected {
    background-color: #5196F5;
    color: #2F3237;
}

/* ========== Status Bar ========== */
QStatusBar {
    background-color: #1C1E21;
    color: #EBEBEB;
}

/* ========== Scroll Bars ========== */
QScrollBar:vertical, QScrollBar:horizontal {
    background: #1C1E21;
}
QScrollBar::handle {
    background: #5196F5;
    border-radius: 4px;
}

/* ========== Table Widget (QTableWidget) ========== */
QTableWidget {
    background-color: #1C1E21;
    alternate-background-color: #26282C;
    color: #EBEBEB;
    gridline-color: #2F3237;
    selection-background-color: #5196F5;
    selection-color: #1C1E21;
    border: 1px solid #2F3237;
}
QHeaderView::section {
    background-color: #2F3237;
    color: #EBEBEB;
    padding: 4px;
    border: 1px solid #26282C;
}
QTableView::item:hover {
    background-color: #3a3d43;
}
QTableView::item:selected {
    background-color: #5196F5;
    color: #1C1E21;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>755</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
    </property>
    <item row="2" column="0">
     <widget class="QFrame" name="sidebarFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::LeftToRight</enum>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0,1,0,1,0,10,0">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QPushButton" name="dashboardButton">
         <property name="text">
          <string>Dashboard</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="Spacer2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="sectorButton">
         <property name="text">
          <string>Sectors Overview</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="Spacer3">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="stocksrelativeButton">
         <property name="text">
          <string>Stock Relative 
Performances</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="Spacer4">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="foundamentalsButton">
         <property name="text">
          <string>Financials  
Analysis</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="settingsSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="settingsButton">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2">
     <widget class="QPushButton" name="collapseSidebarButton">
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="toolTip">
       <string>&quot;Collapse/Expand Sidebar&quot;</string>
      </property>
      <property name="toolTipDuration">
       <number>1</number>
      </property>
      <property name="text">
       <string>â˜°</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QStackedWidget" name="mainContentStackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QWidget" name="mainContentStackedWidgetPage1" native="true">
       <widget class="QTableWidget" name="dashboardTableWidget">
        <property name="geometry">
         <rect>
          <x>35</x>
          <y>11</y>
          <width>991</width>
          <height>541</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1256</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
